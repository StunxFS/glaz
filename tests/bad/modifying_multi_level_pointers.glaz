fn main() {
	let x = 9;
	let y: *i32 = &x;
	let mut v = &y;
	v.*.* = 1;
}
