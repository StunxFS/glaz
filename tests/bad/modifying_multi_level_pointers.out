tests/bad/modifying_multi_level_pointers.glaz:5:3: error: dereference of pointer is unsafe and requires `unsafe` block
     | 
   5 | 	v.*.* = 1;
     | 	^^^^^
glaz error: could not compile `modifying_multi_level_pointers`, aborting due to previous error.
